addon/search/search.js

```diff
@@ -141,6 +141,7 @@
     state.lastQuery = state.query;
     if (!state.query) return;
     state.query = null;
+    state.queryText = null;
```

Comment:
queryText needs to be reset as well.  otherwise on a second search by user the following JS error is thrown

Uncaught TypeError: Cannot read property 'global' of nullSearchCursor 
@ searchcursor.js:27(anonymous function) 
@ searchcursor.js:173getSearchCursor 
@ search.js:57(anonymous function) 
@ search.js:129runInOp 
@ codemirror.js:3126CodeMirror.operation 
@ codemirror.js:5209findNext 
@ search.js:127(anonymous function) 
@ search.js:114(anonymous function) 
@ dialog.js:78